core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -419,6 +419,9 @@ public enum Property {
       PropertyType.COUNT, "Max number of files a major compaction thread can open at once. "),
   TSERV_SCAN_MAX_OPENFILES("tserver.scan.files.open.max", "100", PropertyType.COUNT,
       "Maximum total files that all tablets in a tablet server can open for scans. "),
+  @Experimental
+  TSERV_SCAN_INITIAL_WAIT_ENABLED("tserver.scan.initial.wait.enabled", "true", PropertyType.BOOLEAN,
+      "Determines if the client waits for writes before scanning a table for the first time"),
```

Comment:
Might want to include why.  Something along the lines of:

`Determines if the client waits for writes in progress before scanning a table for the first time to ensure the client sees all data previously written.`